.prod/on_deploy.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+echo "on_deploy.sh here ..."
+
+python /app/manage.py migrate --noinput
+
+# Generate the admin user using the password given in the environment variables.
+# If no password is set, the admin user gets a generated password which will
+# be written in stdout so that it can be accessed during the initial deployment.
+if [[ "$ADMIN_USER_PASSWORD" ]]; then
```

Comment:
I this the admin creation here because we can't use `CREATE_ADMIN_USER` on the `docker-entrypoint.sh` in review/staging/production?

Reply:
I've been wondering about this myself. We can use the env variables and `docker-entrypoint.sh`, unless I'm mistaken.

The current setup is about the same as in Helsinki profile which means that `on_deploy.sh` and `docker-entrypoint.sh` are doing basically the same thing, except that `docker-entrypoint.sh` can be controlled more with env variables.

Initially I thought that Gitlab uses only `on_deploy.sh`, but I think it's also executing `docker-entrypoint.sh`.

Do you have a suggestion on how this should be? Should I just remove the duplicate stuff from `on_deploy.sh` and put suitable env vars (e.g. `CREATE_ADMIN_USER = 1`) in the Gitlab variables so that `docker-entrypoint.sh` is used to migrate the database and create admin/page contents?

Reply:
I think `on_deploy.sh` is the correct place to do migrations on review/staging/production. I think that file is guaranteed to be run only once. I guess theoretically docker-entrypoint might run at the same time on multiple containers (if the service was scaled to run e.g. 3 containers) which might have interesting results if a migration script for that service was started at the same time an all containers.

TL;DR I think `on_deploy.sh` is fine for now. :+1:

Reply:
OK, we'll go with that, thanks!